src/features/events/l10n/messageIds.ts

```diff
@@ -54,6 +54,8 @@ export default makeMessages('feat.events', {
     move: m('Move'),
     moveInstructions: m('Drag the pin to select location.'),
     noDescription: m('No description'),
+    noRelatedEvents: m('There are no other events in this location.'),
+    relatedEvents: m('Other events in this location'),
```

Comment:
Maybe we should rephrase this as _Other events around this time_? It's obvious from the visual context that it relates to a certain location, is it not?

Reply:
I had the same thoughts, that it is not completely clear that it means that they occur simultaneously. Like, it would be easy for me to assum they were just all events at that place no matter the time. I agree with your suggestion!